client/verta/verta/_deployment/endpoint.py

```diff
@@ -213,3 +213,9 @@ def get_access_token(self):
         if len(tokens) == 0:
             return None
         return tokens[0]['creator_request']['value']
+
+    def form_update_body(self, resources, strategy, build_id):
```

Comment:
This seems like something that should be considered private.
```suggestion
    def _form_update_body(self, resources, strategy, build_id):
```

Reply:
made private